turbopack/crates/turbo-tasks/src/output.rs

```diff
@@ -0,0 +1,40 @@
+use std::{
+    borrow::Cow,
+    fmt::{self, Display},
+};
+
+use anyhow::anyhow;
+
+use crate::{util::SharedError, RawVc};
+
+/// A helper type representing the output of a resolved task.
+#[derive(Clone, Debug)]
+pub enum OutputContent {
+    Link(RawVc),
+    Error(SharedError),
+    Panic(Option<Box<Cow<'static, str>>>),
+}
+
+impl OutputContent {
+    /// INVALIDATION: Be careful with this, it will not track dependencies, so
+    /// using it could break cache invalidation.
+    pub fn read_untracked(&self) -> anyhow::Result<RawVc> {
```

Comment:
The name of this method is weird. `untracked` doesn't fit here. Maybe call it `as_read_result()`. And remove the comment